app/src/modules/content/routes/item.vue

```diff
@@ -788,14 +802,7 @@ function useCollectionRoute() {
 					:primary-key="actualPrimaryKey"
 					:allowed="shareAllowed"
 				/>
-				<flow-sidebar-detail
-					v-if="currentVersion === null"
-					location="item"
-					:collection="collection"
-					:primary-key="actualPrimaryKey"
-					:has-edits="hasEdits"
-					@refresh="refresh"
-				/>
+				<flow-sidebar-detail :manual-flows />
```

Comment:
Let’s make sure to keep flows disabled for versions, since this is currently not supported.

```suggestion
				<flow-sidebar-detail v-if="currentVersion === null" :manual-flows />
```